src/management/JobsManager.js

```diff
@@ -189,6 +202,13 @@ JobsManager.prototype.importUsers = function(data, cb) {
             error.status = res.statusCode;
             error.method = method;
             error.text = res.text;
+            try {
+              if (!error.text && res.body) {
+                error.text = JSON.parse(res.body).message;
+              }
+            } catch (ex) {
+              console.error(ex);
```

Comment:
I think there's no need to log anything here. I'm assuming the res.body parsing is optional (assuming that `res.text` will have all the info we need). If that's the case, you can silently ignore the error

Reply:
And if you don't have either `error.text` nor can't parse `res.body`, maybe you can set a default message here to `error.text`? Assuming none is set before this line

Reply:
I went with just ignoring it.  It's really just a secondary attempt to get some useful info.  The error code "should" be there so that is probably the last resort.